### ISSUE TITLE
DBMS_OUTPUT Package

### ISSUE THREAD

**Issue by ahsanhadi** [pgsql-io/pgosql#9]
The DBMS_OUTPUT package enables the user to send messages from stored procedures and packages. It is typically used for debugging PL programs i.e. procedures, packages, triggers etc, it is also used for displaying messages at terminal output for SQL based reports.

**Comment by luss** [comment]
Yup.  We need this and it is essential from day one.  My present
thinking is that by always including OraFCE in our binaries, it will be
there.
